scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -102,7 +105,10 @@ output_sce_file <- file.path(normalized_dir,
 
 ## Reading Cell Ranger data
 
+![Single-cell roadmap preprocessing](diagrams/roadmap_single_preprocess.png)
+
 Whether the 10X Cell Ranger data is in Matrix Exchange format or in an HDF5 file, we can use the `read10xCounts()` function from the `DropletUtils` package to read the data into R and create a `SingleCellExperiment` object.
+(Though again, we do not recommend using the HDF5 file if you can avoid it, _especially_ for raw data.)
```

Comment:
for consistency 
```suggestion
(Though again, we do not recommend using the `HDF5` file if you can avoid it, _especially_ for raw data.)
```

Also wondered if you might want to say "unfiltered" instead of "raw"? Or a parenthetical, `raw (unfiltered) data.`

Reply:
I used "raw" because that is the file/folder is called by Cell Ranger. I actually wrote it with "unfiltered," but then changed it to match Cell Ranger. Parenthetical seems good. 
```suggestion
(Though again, we do not recommend using the `.h5` file if you can avoid it, _especially_ for raw (unfiltered) data.)
```